ui/DesignSystem/Component/ConnectionStatusIndicator.svelte

```diff
@@ -1,20 +1,12 @@
 <script>
-  export let peerState;
```

Comment:
I wonder if this component should not live in `/ui/App/`, because it has data fetching built-in?

Reply:
Yeah it should, but then id shouldn't be imported by the sidebar. It's a bit odd still that the sidebar lives in the design system.

Reply:
We can move that out too.

Reply:
Wouldn't this fix that? https://github.com/radicle-dev/radicle-upstream/pull/1046#discussion_r505229282

Reply:
Yeah, that works even better @NunoAlexandre.